runtime/onert/core/src/util/shapeinf/Reshape.cc

```diff
@@ -91,8 +91,7 @@ void StaticInferer::visit(const ir::operation::Reshape &op)
     const auto *shape_buf = reinterpret_cast<const int32_t *>(shape.data()->base());
     assert(shape_buf);
 
-    assert(shape.shape().rank() == 1);
-    new_shape = convertShape(shape_buf, shape.shape().dim(0), input.shape().num_elements());
+    new_shape = convertShape(shape_buf, shape.shape().num_elements(), input.shape().num_elements());
```

Comment:
Could you also change the name of second param (`const int32_t rank`) of `convertShape()` to align with the real meaning?

Reply:
@hyunsik-yoon Thanks. I've updated.